### PR TITLE
tf13 upgrade and circleci changes

### PR DESCRIPTION
**Pull request by pgvishnuram**
* terraform13 changes
* circleci config update
* precommit yaml upgdates
* script cleanup
* spotinst provider changes
* acme provider version update

### CODE REVIEW

**Comment by pgvishnuram** [comment]
acme version not supported raising new PR with fix

File: .drone.yml [hunk 1]
```diff
@@ -2,153 +2,151 @@ kind: pipeline
 name: terraform-google-astronomer-gcp
```
Comment (danielhoherd) [inline .drone.yml:2]: We should delete this file. It's not going to be useful to us anymore, and we can always dig into git history if we need to reference it.

Comment (pgvishnuram) [reply .drone.yml:2]: okay. Let me update the PR

**Review by danielhoherd — APPROVED** [review]
Looks good!

**Pull request merged**